### PR TITLE
Bug 1428288, Added a section on subPath

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1428288

Preview Build: http://file.rdu.redhat.com/~ahardin/03232017/subPath/dev_guide/volumes.html#volumes-specifying-a-subpath

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@pmorie Here's my first pass at addressing sub-path, based on our conversation. PTAL.

**Comment by ahardin-rh** [comment]
@bmcelvee @mburke5678 @bfallonf Can one of you please peer review? Thanks!

**Comment by bmcelvee** [comment]
LGTM! 🎆

**Comment by bfallonf** [comment]
LGTM :+1:

File: dev_guide/volumes.adoc [hunk 1]
```diff
@@ -306,3 +314,38 @@ List volume *v1* defined on all deployment configurations:
...
 ----
+
+[[volumes-specifying-a-subpath]]
+== Specifying a Sub-path
+
+Use the `volumeMounts.subPath` property to specify a `subPath` inside a volume
+instead of the volume's root. `subPath` allows you to share one volume for
+multiple uses in a single pod.
+
+.Example subPath Usage
```
Comment (pmorie) [inline dev_guide/volumes.adoc]: The only suggestion I have here is add a preamble showing the contents of this volume.

File: dev_guide/volumes.adoc [hunk 2]
```diff
@@ -306,3 +314,51 @@ List volume *v1* defined on all deployment configurations:
...
 ----
+
+[[volumes-specifying-a-subpath]]
+== Specifying a Sub-path
+
+Use the `volumeMounts.subPath` property to specify a `subPath` inside a volume
+instead of the volume's root. `subPath` allows you to share one volume for
+multiple uses in a single pod.
+
+To view the contents of the volume, run `oc volume pod/<pod_name>`. For example:
```
Comment (ahardin-rh) [inline dev_guide/volumes.adoc]: @pmorie Is this what you mean? Showing `oc volume pod`? If so, I can get this example updated to match the Example subPath Usage below. Thanks!

File: dev_guide/volumes.adoc [hunk 3]
```diff
@@ -23,14 +23,14 @@ are present, to repair them when possible, {product-title} invokes the `fsck`
...
-The simplest volume type is `*EmptyDir*`, which is a temporary directory on a
+The simplest volume type is `EmptyDir`, which is a temporary directory on a
 single machine. Administrators may also allow you to request a
 xref:persistent_volumes.adoc#dev-guide-persistent-volumes[persistent volume] that is automatically attached
 to your pods.
 
 [NOTE]
 ====
-`*EmptyDir*` volume storage may be restricted by a quota based on the pods FSGroup, if enabled by your cluster administrator.
+`EmptyDir` volume storage may be restricted by a quota based on the pods FSGroup, if enabled by your cluster administrator.
```
Comment (mburke5678) [inline dev_guide/volumes.adoc]: s/pods/pod's
s/if enabled/if the FSGroup parameter
Do we need a link to enable FSGroup?

File: dev_guide/volumes.adoc (hunk 2 above)
Comment (ahardin-rh) [inline dev_guide/volumes.adoc]: @pmorie bump ^^

Comment (pmorie) [inline dev_guide/volumes.adoc]: I had meant just showing the files within the volume.

**Comment by vikram-redhat** [comment]
@ahardin-rh - can this be moved along?

**Comment by ahardin-rh** [comment]
@vikram-redhat Updates coming soon. I need to rework the example and some other bits. Thanks!

**Pull request merged**